<?xml version="1.0"?>
<robot name="erp42" xmlns:xacro="http://www.ros.org/wiki/xacro">
  <xacro:property name="PI" value="3.1415926535897931" />


<link name="base_link_ys"></link>

  <link name="velodyne_ys"> </link>
  <joint name="velodyne_ys_joint" type="fixed">
    <parent link="base_link_ys" />
    <child link="velodyne_ys" />
    <origin xyz="0.38 0 0" rpy="0 0 0" />
  </joint>
</robot>


